source/class/cv/ui/structure/tile/components/Select.js

```diff
@@ -97,25 +127,47 @@ qx.Class.define('cv.ui.structure.tile.components.Select', {
     },
 
     _updateValue(mappedValue, value) {
-      if (this.__options.has(mappedValue)) {
+      const key = typeof mappedValue !== 'undefined' ? '' + mappedValue : '';
```

Comment:
Would this work also and be better?
```
      const key = '' + (mappedValue ?? '');
```

Reply:
IMHO thats not the same e.g. 0 would be converted to an emtpy string, `false` also. The existing line should convert everything to string but avoid `null` and `undefined` as strings, thats what the type check is for.